tests/test_pagination.py

```diff
@@ -71,6 +62,11 @@ def test_run(self):
         self.TESTABLE_STREAMS = self.testable_streams().difference(self.sandbox_streams())
         self.pagination_test()
 
+        print("\n\n-- SKIPPING -- TESTING WITH STATIC DATA IN SQUARE_ENVIRONMENT: {}".format(os.getenv('TAP_SQUARE_ENVIRONMENT')))
```

Comment:
Shouldn't this no longer say skipping?

Reply:
The only static stream is `locations` now and that doesn't paginate. So I will remove the references and put a comment explaining that.